pokerole_irl/user_app/models.py

```diff
@@ -11,3 +11,25 @@ class Profile(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Setting(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Do we want to CASCADE here? If a user is deleted, do we automatically delete the setting, or do we rather want to leave the setting around without a user? Maybe (in the future) implement a way to "claim" an ownerless setting?

Reply:
Changed to SET_NULL